src/components/CommentForm.vue

```diff
@@ -0,0 +1,89 @@
+<script setup>
+import { ref, onMounted, watch } from 'vue'
+import { addComment } from '@/api/comments'
+import InputField from './InputField.vue'
+import TextAreaField from './TextAreaField.vue'
+
+const { postId, formName, formEmail } = defineProps({
+  postId: Number,
+  formName: String,
+  formEmail: String
+})
+
+const emits = defineEmits(['closeForm', 'addComment'])
+
+const name = ref('')
+const email = ref('')
+const body = ref('')
+
+const errors = ref({
+  name: '',
+  email: '',
+  body: ''
+})
```

Comment:
You have several ref variables (name, email, body, errors). Using ref is fine, but when managing forms, using `reactive` for the form data and errors is often more convenient since it groups related state into a single object, especially when you have more fields.